README.md

```diff
@@ -37,6 +41,31 @@ b = Dagger.@spawn rand(a, 4)
 c = Dagger.@spawn sum(b)
 fetch(c) # some number!
 ```
+## Contributing Guide
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+![GitHub issues](https://img.shields.io/github/issues/JuliaParallel/Dagger.jl)
+![GitHub contributors](https://img.shields.io/github/contributors/JuliaParallel/Dagger.jl)
+[![GitHub](https://badgen.net/badge/icon/github?icon=github&label)](https://github.com)
+
+Contributions are encouraged. 
+
+There are several ways to contribute to our project:
+
+**Reporting Bugs**: If you find a bug, please open an issue and describe the problem. Make sure to include steps to reproduce the issue and any error messages you receive regarding that issue.
+
+**Fixing Bugs**: If you'd like to fix a bug, please create a pull request with your changes. Make sure to include a description of the problem and how your changes will address it.
+
+Additional examples and documentation improvements are also very welcome.
+
+## Documentation
+List of recommended Dagger resources:
+- https://docs.juliahub.com/Dagger/0a2f8/0.13.3/
```

Comment:
```suggestion
- Docs [![][docs-master-img]][docs-master-url]
```

Since otherwise we're pointing to a specific, old commit's docs.

Reply:
done!